realm/realm-library/src/objectServer/java/io/realm/permissions/PermissionChange.java

```diff
@@ -18,6 +18,7 @@
 import java.util.Date;
 import java.util.UUID;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
```

Comment:
http://stackoverflow.com/questions/37422191/how-to-remove-suppressfbwarnings-annotations-from-released-jar-s